client/pokemon/pokemon.ctrl.js

```diff
@@ -54,6 +56,8 @@ module.exports = class Pokemon {
     this.isVC = this.data.otGameId >= 35 && this.data.otGameId <= 38;
     this.hasHA = this.data.abilityNum === 4;
 
+    this.originPre7 = this.data.otGameId < 30;
```

Comment:
Good point. Alright, I'll change it so that the VC games show neither sid nor tidfull, because the sid is blank, and tidfull is the same as idno.